ast.go

```diff
@@ -1882,7 +1882,17 @@ func (node *OnConflictClause) walkSubtree(visit Visit) error {
 	if node == nil {
 		return nil
 	}
-	return Walk(visit, node.Target.Columns, node.Target.Where, node.DoUpdate.Exprs, node.DoUpdate.Where)
+
```

Comment:
`Target` and `DoUpdate` can be `nil`. Gotta check for that

Reply:
👍